modules/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -148,6 +148,9 @@ error.incompatible.types.cast=\
 error.invalid.action.invocation=\
   invalid action invocation on connector: ''{0}'', expect endpoint
 
+error.invalid.function.invocation=\
+  function invocation on {0} not supported
```

Comment:
shall we reword "...{0} not supported" --> "... type ' '{0}' ' is not supported"